_posts/2020-03-06-introducing-mirrormaker2.md

```diff
@@ -0,0 +1,268 @@
+The _heartbeat_ topic is used to check that the source cluster is available and the clusters are connected.
+
+### Unleashing MirrorMaker 2.0
+
+Okay, so let's look at how you might approach the configuration of a MirrorMaker 2.0 deployment with the `KafkaMirrorMaker2` resource.
+
+When you define your MirrorMaker 2.0 configuration to set up a connection, you specify the `sourceCluster` and `targetCluster`, and the `bootstrapServers` for connection.
+
+If you're sticking with defaults, you don't need to specify much more.
+Here's a minimum configuration for MirrorMaker 2.0:
+
+```yaml
+apiVersion: kafka.strimzi.io/v1alpha1
+kind: KafkaMirrorMaker2
+metadata:
+  name: my-mirror-maker2
+spec:
+  version: 2.4.0
+  connectCluster: "my-cluster-target"
+  clusters:
+  - alias: "my-cluster-source"
+    bootstrapServers: my-cluster-source-kafka-bootstrap:9092
+  - alias: "my-cluster-target"
+    bootstrapServers: my-cluster-target-kafka-bootstrap:9092
+  mirrors:
+  - sourceCluster: "my-cluster-source"
+    targetCluster: "my-cluster-target"
+    sourceConnector: {}
+```
+
+Note that the `spec` includes the Kafka Connect version and cluster alias with the `connectCluster` value,
+as we're using a Kafka Connect framework to make our connection.
+
+The `clusters` properties define the Kafka clusters being synchronized, and the `mirrors` properties define the MirrorMaker 2.0 connectors.
+
+You can build a more elaborate configuration.
+For example, you can add configuration to include replica nodes, which is the number of replica nodes in the Kafka Connect group, and TLS or SASL authentication for the source and target cluster too.
+Typical and sensible choices.
+
+Here we create 3 replica nodes and use TLS authentication.
+
+```yaml
+apiVersion: kafka.strimzi.io/v1alpha1
+kind: KafkaMirrorMaker2
+metadata:
+  name: my-mirror-maker2
+spec:
+  version: {DefaultKafkaVersion}
+  replicas: 3
+  connectCluster: "my-cluster-target"
+    clusters:
+    - alias: "my-cluster-source"
+    authentication:
+      certificateAndKey:
+        certificate: source.crt
+        key: source.key
+        secretName: my-user-source
+      type: tls
+    bootstrapServers: my-cluster-source-kafka-bootstrap:9092
+    tls:
+      trustedCertificates:
+      - certificate: ca.crt
+        secretName: my-cluster-source-cluster-ca-cert
+  - alias: "my-cluster-target"
+    authentication:
+      certificateAndKey:
+        certificate: target.crt
+        key: target.key
+        secretName: my-user-target
+      type: tls
+    bootstrapServers: my-cluster-target-kafka-bootstrap:9092
+    config:
+      config.storage.replication.factor: 1
+      offset.storage.replication.factor: 1
+      status.storage.replication.factor: 1
+    tls:
+      trustedCertificates:
+      - certificate: ca.crt
+        secretName: my-cluster-target-cluster-ca-cert
+```
+
+People can be confused by the idea of _replicas_ and _replication_ through MirrorMaker,
```

Comment:
Given you talked abotu replicas just above the example as Kubernetes replcias, do we need to clarify that here as well?

Reply:
Changed to:

> _People can be confused by the idea of _replicas_ and _replication_ through MirrorMaker,
> that's why you'll see replication between clusters often referred to as _mirroring_ so that it's not confused with 
> the `replicas` that represent the nodes replicated in a deployment._